docker-compose.yml

```diff
@@ -1,13 +1,48 @@
-version: '3'
+# Version 3, most current, and recommended compose file version
+# https://docs.docker.com/compose/compose-file/compose-versioning/
+version: "3"
+
 services:
+
+  # First service
   telescope:
-    build: .
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file: .env
+    environment:
+      # a place-holder for the variable passed by CLI and with a default value
+      # the passed or default value is the desired command to run `telescope`
+      script: ${script:-debug-server}
     depends_on:
       - redis
-    ports:
-      - '3000:3000'
+    # PORT MAPPING IN `host` MODE IS WRONG AND MUST BE CHANGED.
```

Comment:
If there's a follow-up issue to deal with here, let's file that instead of this.

Reply:
It gets solved actually by @manekenpix

Reply:
OK, I wondered, but since you'd left this in, I wasn't sure.  We can remove this then.

Reply:
@manekenpix solved it today actually, while I posted this last night as a draft, mostly because @Grommers00 asked me so.